networkx/linalg/laplacianmatrix.py

```diff
@@ -146,6 +147,40 @@ def normalized_laplacian_matrix(G, nodelist=None, weight="weight"):
     return sp.sparse.csr_matrix(DH @ (L @ DH))
 
 
+def total_spanning_tree_weight(G, weight=None):
+    """
+    Apply Kirchhoff's Tree Matrix Theorem to a graph in order to find the total
+    weight of all spanning trees.
+
+    The theorem states that the determinant of any cofactor of the Laplacian
+    matrix of a graph is the number of spanning trees in the graph. For a
```

Comment:
Making the first line shorter improves readability of the hyperlinks in the documentation to this function.
```suggestion
    Returns the total weight of all spanning trees of `G`.
        
    Kirchoff's Tree Matrix Theorem states that the determinant of any cofactor of the 
    Laplacian matrix of a graph is the number of spanning trees in the graph. For a
```